lsif/src/shared/xrepo/commits.ts

```diff
@@ -122,7 +122,28 @@ export async function gitserverExecLines(gitserverUrl: string, repository: strin
  * @param args The command to run in the repository's git directory.
  */
 async function gitserverExec(gitserverUrl: string, repository: string, args: string[]): Promise<string> {
-    const body = JSON.stringify({ repo: repository, args })
-    const resp = await got(new URL(`http://${gitserverUrl}/exec`).href, { body })
+    if (args.length > 0 && args[0] === 'git') {
```

Comment:
This is fine in JS :)
```suggestion
    if (args[0] === 'git') {
```